CONTRIBUTING.md

```diff
@@ -48,6 +48,14 @@ _Before running this command, make sure Docker is running. See [Visual testing](
 
 ---
 
+### Using GitHub Codespaces (cloud IDE)
+
+To get started without having to install anything locally, you can create a [codespace](https://docs.github.com/en/codespaces/overview) for this repository by clicking this link:
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/iTwin/iTwinUI)
+
+---
```

Comment:
This is in the wrong place. It should be directly under "## How to setup" (~L17)

Other suggested changes:
- Add a heading "### Local setup" directly below it.
- Remove the "VSCode users" heading, turn it into static bold text.
- Rename "## Run locally" to "### Commands", and change all sub-headings inside it to be h4 (####)

![image](https://github.com/iTwin/iTwinUI/assets/9084735/e931f89e-938b-421e-b13c-94d6cae07c73)

Reply:
changes made

Reply:
sorry made those changes before the comment edit. I'll make those now